app/lib/pre_assembly/from_staging_location/file.rb

```diff
@@ -52,10 +51,10 @@ def file_attributes
 
       private
 
-      attr_reader :file, :processing_configuration
+      attr_reader :file, :ocr_available
 
       def file_attributes_for_mimetype(mimetype)
-        return ATTRIBUTES_FOR_TYPE_WITH_OCR[mimetype] if processing_configuration == :filename_with_ocr && ATTRIBUTES_FOR_TYPE_WITH_OCR.key?(mimetype)
+        return ATTRIBUTES_FOR_TYPE_WITH_OCR[mimetype] if ocr_available && ATTRIBUTES_FOR_TYPE_WITH_OCR.key?(mimetype)
 
```

Comment:
whether or not we have OCR is now determined by a radio button in the UI and NOT the `filename (with ocr available)` content structure, which is now gone.... so we need that new radio button here instead of the procesisng configuration